app/src/main/java/edu/uc/it3048/mypokedex/ProfileScreenActivity.kt

```diff
@@ -18,8 +18,8 @@ import java.util.jar.Manifest
 
 class ProfileScreenActivity : AppCompatActivity() {
 
-    private val LOGIN_REQUEST_CODE: Int = 607
```

Comment:
Variables should be camel-cased with no underscores - https://kotlinlang.org/docs/reference/coding-conventions.html#formatting

Reply:
This looks like a constant, where uppercase and underscores are permitted, so I'd just add the "const" modifier to this declaration.